Sources/ProjectDescription/BuildRule+CompilerSpec.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+
+extension BuildRule {
+    public enum CompilerSpec: Codable {
```

Comment:
Is there some apple documentation for this? If yes, I would link it. Same for FileType

Reply:
I don't think so. I had to do it the hard way by writing down all options from xcode settings and pbxproj.